src/utils/github/queries/data.ts

```diff
@@ -0,0 +1,149 @@
+//#region > Imports
+//> GQL
+// DocumentNode needed for queries
+import { gql } from "apollo-boost";
+//#endregion
+
+//#region > Queries
+const profile = () => gql`
+  query profile($username: String!) {
+    user(login: $username) {
+      avatarUrl
+      company
+      createdAt
+      name
+      login
+      email
+      websiteUrl
+      hovercard {
+        contexts {
+          message
+          octicon
+        }
+      }
+      isEmployee
+      isHireable
+      location
+      status {
+        emojiHTML
+        message
+      }
+      organizations(first: 100) {
+        edges {
+          node {
+            name
+            login
+            url
+            avatarUrl
+            name
+            description
+            membersWithRole(first: 100) {
+              totalCount
+              nodes {
+                name
+                login
+                avatarUrl
+                url
+                projectsUrl
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+`;
+
+const calendar = (fragments: string) => gql`
+  query calendar ($username: String!){
+    user(login: $username) {
+      current: contributionsCollection {
+        totalIssueContributions
+        totalCommitContributions
+        totalRepositoryContributions
+        totalPullRequestContributions
+        totalPullRequestReviewContributions
+        totalRepositoriesWithContributedIssues
+        totalRepositoriesWithContributedCommits
+        totalRepositoriesWithContributedPullRequests
+      }
+      ${fragments}
+    }
+  }
+`;
+
+// Fragments
```

Comment:
Remove this comment or add comments to the other queries as well.
```suggestion
```

Reply:
This should be a heading. 
```suggestion
//> Fragments
```